tools/build-tools/src/common/monoRepo.ts

```diff
@@ -8,9 +8,31 @@ import * as path from "path";
 import { execWithErrorAsync, rimrafWithErrorAsync, existsSync, readJsonSync } from "./utils";
 
 export enum MonoRepoKind {
```

Comment:
We should probably document this and its flags. If we have docs elsewhere that break down the release groups, we could link to them here.